src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -33,48 +32,27 @@ const SignOrExecuteForm = (props: SignOrExecuteProps): ReactElement => {
   const canExecute = isCorrectNonce && (props.isExecutable || isNewExecutableTx)
   const willExecute = (props.onlyExecute || shouldExecute) && canExecute
 
-  // Estimate gas limit
-  const { gasLimit, gasLimitError } = useGasLimit(willExecute ? safeTx : undefined)
-  const [advancedParams, setAdvancedParams] = useAdvancedParams(gasLimit)
-
-  // Error
-  const error = safeTxError || gasLimitError
-
   return (
     <>
       {props.children}
 
-      <TxSimulation
-        canExecute
-        gasLimit={advancedParams.gasLimit?.toNumber()}
-        transactions={safeTx}
-        disabled={!!gasLimitError}
-      />
-
       <DecodedTx tx={safeTx} txId={props.txId} />
 
+      <TxSimulation canExecute={canExecute} disabled={false} transactions={safeTx} />
```

Comment:
Not passing the gasLimit to the simulation will mean that it will fall back to the latest block gasLimit and won't catch failures related to it anymore. Imo not an issue since we already show an error from the estimation if gasLimit is too low.